examples/kubernetes-yaml/splunk-otel-collector-gateway.yaml

```diff
@@ -180,9 +178,9 @@ spec:
           - "--config=/conf/otel-collector-config.yaml"
           # TODO: Ensure this is set properly
 #           Memory Ballast size should be max 1/3 to 1/2 of memory.
-          - "--mem-ballast-size-mib=683"
+          - "--mem-ballast-size-mib=650"
     # TODO: Ensure this is set properly
-    image: quay.io/signalfx/splunk-otel-collector:0.20.0
+    image: quay.io/signalfx/splunk-otel-collector:latest
```

Comment:
I understand _why_ we would use latest here but I do worry that some will take :latest and use it in their prod deployments, which ain't great for stability.

Reply:
yeah, good call -- I will add a comment about this